src/main/java/core/basesyntax/Application.java

```diff
@@ -1,7 +1,13 @@
 package core.basesyntax;
 
+import core.basesyntax.supplier.Lottery;
+
 public class Application {
     public static void main(String[] args) {
-        // create three balls using class Lottery and print information about them in console
+        Lottery lottery = new Lottery();
+        for (int i = 0; i < 3; i++) {
+            System.out.println(lottery.getRandomBall().toString());
+        }
     }
 }
+
```

Comment:
I opened your solution on my laptop and I see here two empty lines on the 13th and 14th lines, so you can remove one of them)